picard.go

```diff
@@ -1116,7 +1172,16 @@ func getValueFromLookupString(value reflect.Value, lookupString string) reflect.
 }
 
 func getObjectProperty(value reflect.Value, lookupString string) string {
-	return getValueFromLookupString(value, lookupString).String()
```

Comment:
Ditto to above - we can no longer assume that all foreign key fields will be basic string fields, since ActionID and RestObjectID on HeaderValues and ParameterValues are string pointer fields